### DISCUSSION TITLE
Skip HandBrake / Encoding

### DISCUSSION THREAD

**Discussion by trleith** [automatic-ripping-machine/automatic-ripping-machine#811] [Q&A] [answered]
I'm getting the impression MakeMKV can rip losslessly. I want to try it in ARM.

Is there a way to skip the HandBrake / encoding step entirely and keep the MakeMKV output as "completed".

**Comment by 1337-server** [comment, marked as answer]
Have a look through the arm.yaml it has a lot of custom settings to change the arm logic.

`SKIP_TRANSCODE: true`
`DELRAWFILES: false`

Should do what you are wanting

**Reply by trleith** [reply]
Thanks, that seems to work.
